### DISCUSSION TITLE
Setting SMTP from name

### DISCUSSION THREAD

**Discussion by mgbell** [outline/outline#3782] [Ideas]
Is it possible to set the "from" name that appears in emails sent via SMTP? 

If for example I'm using alice@example.com as my SMTP email, but I'd like the name to be shown as bob@, can this be done by setting a variable, something like SMTP_FROM_NAME?

**Comment by tommoor** [comment]
You can use the mailbox format `John Doe <john@example.com>` in `SMTP_FROM_EMAIL`

**Reply by mgbell** [reply]
Thanks Tom, I'll give this a go later today.

**Reply by mgbell** [reply]
Worked perfectly, thank you!
Might I please suggest this is added to the official documentation, or at the least as a comment above the SMTP_FROM_EMAIL variable?

**Reply by tommoor** [reply]
Done 👍  – [Outline SMTP docs](https://wiki.generaloutline.com/share/770a97da-13e5-401e-9f8a-37949c19f97e/doc/smtp-cqCJyZGMIB)

**Reply by mgbell** [reply]
Wow, really glad to see SMTP being properly documented, cheers dude.
